docs/advanced_applications/MemeCam.md

```diff
@@ -0,0 +1,96 @@
+---
+sidebar_position: 5
+---
+
+# 🟡 Reverse engineering MemeCam
+
+[MemeCam](https://www.memecam.io/) is very popular AI-driven app in Reddit recently. 
+It's a web-app that use your camera to photo and wait a second, the app will generate
+a Meme sentence matched with the photo you take and composite the words and photo together.
+Tahda! A (future) viral meme pic generated!
+
+![memecam_example](https://user-images.githubusercontent.com/30429387/231639519-9ec9dbf1-7962-49f4-b694-34b3af450b37.png)
```

Comment:
@trigaten I'm sorry, I can't follow your words. I'm confused because I'm not familiar with GitHub🥲. Do you mean upload the images to this conversation right here below as I am doing rn?
If i'm wrong, please instruct me. thank you so much 😀
<img width="1000" alt="memecam_chinese" src="https://user-images.githubusercontent.com/30429387/231782877-3baf12a5-fdff-4534-8c94-d2e777f98ed7.png">
![memecam_example](https://user-images.githubusercontent.com/30429387/231782897-9423fcfc-3443-4b81-b10c-885db7707edf.png)
![memecam_meme_created](https://user-images.githubusercontent.com/30429387/231782904-f5190c0c-b34f-40cd-a6f7-5ac4ef20e4f2.png)
![memecam_pic_original](https://user-images.githubusercontent.com/30429387/231782915-4d4cd278-407c-4981-8613-f37787fd6d64.jpeg)
<img width="1000" alt="memecam_rev_prompt" src="https://user-images.githubusercontent.com/30429387/231782920-901b0a12-10f4-4a5e-a487-d6a6af5a14ab.png">

Reply:
Sorry, I mean upload them as files to Github in the assets section: https://github.com/trigaten/Learn_Prompting/tree/main/docs/assets

Also, could you 1) move this to applied prompting and 2) make this a bit more robust (e.g. clearly defined sections). I can improve the English later!

Reply:
sure. I think I get you!  Let me do it.